specification/communication/data-plane/Chat/stable/2021-09-07/communicationserviceschat.json

```diff
@@ -1688,6 +1700,48 @@
           "example": "Bob Admin"
         }
       }
+    },
+    "RetentionPolicy": {
+      "description": "Data retention policy for auto deletion.",
+      "type": "object",
+      "discriminator": "retentionPolicyType",
+      "properties": {
+        "retentionPolicyType": {
+          "description": "Retention Policy Type",
+          "enum": [
+            "basedOnThreadCreationDate"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "RetentionPolicyType",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "retentionPolicyType"
+      ]
+    },  
+    "BasedOnThreadCreationDateRetentionPolicy": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/RetentionPolicy"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "daysAfterCreation":{
+              "type": "integer",
+              "format": "int32",
+              "description": "Indicates how many days after the thread creation the thread will be deleted. Only 90 is accepted for now."
```

Comment:
>how many days after
is the thread going to be deleted at the beginning of the 91st day? or at the same time it was created? (could we make it explicit in the API description?)